internal/raft/replicator_test.go

```diff
@@ -44,6 +45,33 @@ func testListMembers(t *testing.T) {
 	clus.assertMembers(t, members)
 }
 
+func testSaveLoadLargeData(t *testing.T) {
```

Comment:
Given that its not very clear if this test is verifying whether snap/wal purge is actually happening, would you like to rewrite this test to verify `purgeFile` more directly ? Note that tests can access private functions inside the same package.

Reply:
This wasn't meant for purgeFile, but more for ensuring that writes continue even after `Snapshot` takes place, which would otherwise fail if there is any error in that logic